src/test/utils.js

```diff
@@ -46,7 +47,9 @@ loadModels(store)
 export const withAppContext = (Component) => (
   <ThemeProvider>
     <Provider store={store}>
```

Comment:
Maybe, for another story, would be good to rename this Provider, like ReduxProvider cause thats what it is

Reply:
Made a jira story for it.